PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,16 @@
-<!-- Please describe your changes on the following line: -->
+**REMOVE:** Thank you for contributing to Servo! This template text contains instructions for the pull request author that should not be part of the final pull request description.
+Please follow these instructions and ensure such lines marked with **REMOVE:** are deleted when creating the pull request.
```

Comment:
Purely from experience, since L2-4 does not start with `**REMOVE:**` some very confused folks will potentially ignore them.

But ofc marking all of them with `**REMOVE:**` is very visually unpleasing. Personally I would just add more required checkboxes say like:
- [ ] I've removed all lines marked starts with `**REMOVE THIS**`, I understand not doing so will require additional verification as part of the review process
- [ ] I've made sure that "Allow edits from maintainers" checkbox is checked

Which should make the formatting a lot more pleasing.

Reply:
Right, in the previous version [87e8c77](https://github.com/servo/servo/commit/87e8c77a840414d98c178493d036acd537a59b93) I had it that way, but in the rendered markdown view they are part of the same paragraph. If we want to have it on each line, then we can add forced line breaks.